lldb/source/Target/Process.cpp

```diff
@@ -2830,6 +2830,9 @@ Status Process::LoadCore() {
           "Did not get stopped event after loading the core file.");
     }
     RestoreProcessEvents();
+    // Since we hijacked the event stream, we will have we won't have run the
+    // stop hooks.  Make sure we do that here:
+    GetTarget().RunStopHooks(true /* at_initial_stop */);
```

Comment:
Nit: clang-tidy and other tools recognize the `/*argument_name=*/` pattern, which is also what most of lldb is already using. 
```suggestion
    GetTarget().RunStopHooks(/*at_initial_stop=*/true);
```

Reply:
done